gamemodes/terrortown/gamemode/server/sv_roleselection.lua

```diff
@@ -394,16 +612,16 @@ end
 -- @note This function modifies the given `plys` var.
 --
 -- @param table plys The players that can receive roles.
--- @param table roleData The @{ROLE} object of the considered role.
+-- @param int subrole The @{ROLE} index of the considered role.
```

Comment:
Is there an `int` type?

Reply:
https://github.com/TTT-2/TTT2/blame/51be3cb2c75587d9affdaa3a8d8145d3909f5a7f/gamemodes/terrortown/gamemode/server/sv_network_sync.lua#L850, but seems to be wrong, yea.

Reply:
Done with https://github.com/TTT-2/TTT2/pull/649/commits/f26eb2e183979dfd5d55f2660101db70d14b766c